Changing and Adding Properties/changing_properties.js

```diff
@@ -0,0 +1,29 @@
+function play(name, isTurn) {
+	if (isTurn) {
+		console.log(`${name} is now playing!`);
+	}
+}
+
+const player1 = {
+	name: 'Ashley',
+	color: 'purple',
+	isTurn: true,
+	play() {
+		play(this.name, this.isTurn);
+	},
+};
+
+const player2 = {
+	name: 'Guil',
+	color: 'red',
+	isTurn: false,
+	play() {
+		play(this.name, this['isTurn']);
+	},
+};
+
+player1.isTurn = true;
+player2['isTurn'] = true;
+
+player1.play();
+player2.play();
```

Comment:
Also, for the `play` prop, shorthand property syntax would allow you to define an object with property names that match the variable names you want to use as values:
```
const player1 = {
  name: 'Ashley',
  color: 'purple',
  isTurn: true,
  play,
};

const player2 = {
  name: 'Guil',
  color: 'red',
  isTurn: false,
  play,
};
```

Reply:
Thanks, @lycoangelo, This is now updated.